src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -158,6 +159,39 @@ class DevTask extends AbstractServerTask {
         this.pollingTest = pollingTest;
     }
 
+    private Boolean container = null;
+
+    @Option(option = 'container', description = 'Run the server in a Docker container instead of locally. The default value is false for the libertyDev task, and true for the libertyDevc task.')
```

Comment:
I'm very confused why this `private Boolean container` is defined on 3 different tasks so far. And then this is the second time I have seen `@Option(option = 'container'` with a `setContainer` method.

Reply:
`@Option(option = 'container'` can decorate the private variable but it can also decorate the `setContainer` method. We use the set method to pass the option to the super class for use in a method there. 
It is defined multiple times because the dev task calls the deploy task and needs to pass the `container` option. The subtask e.g. deploy runs in a separate JVM so it would seem an option is the clearest way to pass the value.

Reply:
We've removed the extra booleans and changed it to use `project.liberty.dev.container` as suggested in Matt's comment below.